public/scripts/openai.js

```diff
@@ -402,6 +413,24 @@ function validateReverseProxy() {
         resultCheckStatus();
         throw err;
     }
+    const rememberKey = `Proxy_SkipConfirm_${getStringHash(oai_settings.reverse_proxy)}`;
```

Comment:
I think this should only appear when called from getStatusOpen. Otherwise, it will nag you on every prompt.

Reply:
Only if you tick the checkbox

Reply:
Isn't that how it should be? If not, then just remove the Checkbox and also save it as accepted if you click the allow option. No need for the checkbox then. One time popup and confirming it should be enough?

Reply:
> One time popup and confirming it should be enough

Okay, that sounds reasonable. Security is important, but I don't want to be obnoxious.